llvm/include/llvm/DebugInfo/DWARF/DWARFAcceleratorTable.h

```diff
@@ -543,6 +543,19 @@ class DWARFDebugNames : public DWARFAcceleratorTable {
       return StrData.getCStr(&Off);
     }
 
+    /// Compares the name of this entry against Target, returning true if they
+    /// are equal. This is helpful is hot code paths that do not need the length
```

Comment:
Did you mean "This is helpful in" ?

Reply:
Yup! I updated the comment after Adrian's other suggestion